src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -3,7 +3,9 @@
 
 
 {% block header_image %}
+<aside aria-label="{% trans "Header afbeelding" %}">
```

Comment:
Is there any reason these kinds of labels should be explicitly available to assistive technology. In other words: does it add value to for screen readers to read that this section shows only an image?

If this section is purely decorative, I think aria-hidden may be a better choice?

> Adding aria-hidden="true" to an element removes that element and all of its children from the accessibility tree. This can improve the experience for assistive technology users by hiding:

 > Purely decorative content, such as icons or images
 > Duplicated content, such as repeated text
 > Offscreen or collapsed content, such as menus

 > The presence of the aria-hidden attribute hides content from assistive technology but doesn't visually hide anything.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

Reply:
applied aria-hidden